tools/distpartitioning/test_map.py

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Do not upload file that is not relevant.

Reply:
Agreed. Removed. This was some debug file I used for my local unit testing. Removed this file from getting into the codebase.